e2e/page-objects/header.po.ts

```diff
@@ -10,16 +10,16 @@ export class HeaderPage {
   userMenu = element(by.css('.navbar-user'));
   logout = element(by.css('.logout'));
 
-  getElement (name) {
+  getElement (name, getChild) {
```

Comment:
what about keeping the names the same in both the scenario and the page object and doing something like this?

```
  megaMenu = element(by.css('.dropdown-lg'));
  megaMenu.child = element(by.css('.dropdown-header'));
```

```
return getChild ? this[name].child : this[name];
```